### DISCUSSION TITLE
Is `Microsoft.SemanticKernel.Connectors.AI.OpenAI` still in use?

### DISCUSSION THREAD

**Discussion by rushjam** [microsoft/semantic-kernel#6481] [Q&A] [answered]
I want to use tokenizer like GPT3Tokenizer to count number of tokens. But I don't see the library being maintained., last update was 6 months ago. 

Do we have any other library that I can use it do the same job done?

**Comment by stephentoub** [comment, marked as answer]
The library was renamed to Microsoft.SemanticKernel.Connectors.OpenAI.

However, the GPT3Tokenizer type was removed. Use the Microsoft.ML.Tokenizers library for tiktoken tokenization.
